invenio_records_rest/errors.py

```diff
@@ -195,8 +195,14 @@ def __init__(self, error=None, **kwargs):
         """Initialize exception."""
         super(RESTValidationError, self).__init__(**kwargs)
         self.errors = kwargs.get('errors')
-        self.description = 'Validation error: {0}.'.format(
-            error.message if error else '')
+        if self.errors:
```

Comment:
isn't the response json? Can we just have the object with all instead of joining them?